test-cases/gemini/gemini-basic-3h.yaml

```diff
@@ -22,5 +22,3 @@ gemini_schema_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/schema.json'
 
 db_type: mixed_scylla
 instance_type_db_oracle: 'i3.8xlarge'
-oracle_scylla_version: '4.3.3'
-append_scylla_args_oracle: '--enable-cache false'
```

Comment:
same

Reply:
this case wasn't run for almost a year now, it shouldn't be pinning specific version, `test_default.yaml` has the version all the gemini tests uses